rust/lance/src/dataset/scanner.rs

```diff
@@ -627,7 +629,24 @@ impl Scanner {
             legacy_with_row_id: false,
             explicit_projection: false,
             autoproject_scoring_columns: true,
-        }
+        };
+        scanner.apply_blob_handling();
+        scanner
+    }
+
+    fn apply_blob_handling(&mut self) {
+        let projection = self
+            .projection_plan
+            .physical_projection
+            .clone()
+            .with_blob_handling(self.blob_handling.clone());
+        self.projection_plan.physical_projection = projection;
+    }
```

Comment:
Do we also need to call `apply_blob_handling` in `Scanner::project_with_transform`?

Reply:
Yep, we already did that 😆 

https://github.com/lancedb/lance/pull/5189/files#diff-c25e7065f1e84cc2421b1d8b920d96038c14557fb3daadd07932f719098bc81bR732